### ISSUE TITLE
Prestashop 1.7.5 Specific price and multishop

### ISSUE THREAD

**Issue by sandra2n** [PrestaShop/PrestaShop#12424]
Hello,
In Prestashop 1.7.5.0, I am in multishop config with two shops created. I have a specific price on one of my product, created for one of my shop. But when I am in "All shops" view, I still see this specific price. It would be nice to either, show it only in relevant shop or indicate the concerned shop in the specific price information (by adding a column, maybe?)

Thanks for your help

**Comment by khouloudbelguith** [comment]
Hi @sandra2n,

Thanks for your report.
When "All shops" is selected, in the Pricing tab, if you click to edit the Specific price conditions, you can find that shop is indicated to which shop.
![image](https://user-images.githubusercontent.com/16067358/52205266-c6519c80-2876-11e9-860d-72c1963d50cd.png)
It could be an improvement issue to add a column on the product page, which shop is selected.
![image](https://user-images.githubusercontent.com/16067358/52205351-14ff3680-2877-11e9-9b5f-2684f0848cd3.png)
@marionf @colinegin what do you think?
Thanks!

**Comment by sandra2n** [comment]
Yes, adding a column to see what shop has been choose, could be nice.
Thank you

**Comment by colinegin** [comment]
It's a great idea ! I add it to our to do list.

Thank you

**Comment by Hlavtox** [comment]
Closing as the old product page is gone now. Implemented on the new page.
